packages/app/src/components/AppNotificationManager/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+import { updateAppActions } from '../../actions';
+import { StyledNotificationContainer } from './styled';
+import Notification from './notification';
+import uniqueId from './utils/uniqueId';
+
+const AppNotificationManager = () => {
+    const [notifications, setNotifications] = useState([]);
+
+    const removeNotification = (notification) => {
+        setNotifications((notifications) => {
+            return notifications.filter((item) => {
+                return item.notification !== notification;
+            });
+        });
+    };
+
+    const addNotification = (notification, timeout) => {
```

Comment:
useCallback

Reply:
Adding `useCallback` here does not have any benefits, I ran the profiler on the component and rendering times are bigger when using `useCallback`

Reply:
I ran performance tests on the component with and without `useCallback` and it turned out that is faster without it, so I think that even if we update the app actions object on each render still is more efficient that calling `useCallback`